src/functionalTest/resources/scenarios/RIA-3861-out-of-country-non-sponsor-client-left-country.json

```diff
@@ -0,0 +1,39 @@
+{
+  "description": "RIA-3861 Out of country no sponsor client left country",
+  "launchDarklyKey": "out-of-country-feature:true",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "startAppeal",
+      "state": "appealStarted",
+      "caseData": {
+        "template": "minimal-appeal-started.json",
+        "replacements": {
+          "appellantInUk": "No",
+          "hasCorrespondenceAddress": "Yes",
+          "appellantOutOfCountryAddress": "Kabul, Afghanistan",
+          "hasSponsor": "No",
+          "decisionLetterReceivedDate":"{$TODAY-7}",
```

Comment:
dateClientLeaveUk should be used in here, not decisionLetterReceivedDate

Reply:
Test corrected with dateClientLeaveUk